flycs_sdk/query_base_schema.py

```diff
@@ -0,0 +1,191 @@
+"""This module contains base class for any "query" object that needs to define a schema (Transformation, views)."""
+
+from abc import abstractmethod
+from typing import List, Optional
+
+from flycs_sdk.query_base import QueryBase
+
+BQ_DATA_TYPES = {
```

Comment:
NUMERIC has alias DECIMAL and BIGNUMERIC has alias BIGDECIMAL. Might be interesting to also add them, because we also have all the aliases of INT64 (INT, INTEGER, SMALLINT, ....)

Reply:
yeah, the idea is that we cover every single type supported by BigQuery. If there is any other I forget here, please let me know.